Apps/WebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -277,3 +280,9 @@ export interface ILogger {
 export interface IStoreProvider {
     getStore(): Store<RootState>;
 }
+
+export interface ITicketService {
+    createTicket(room: string): Ticket;
+    removeTicket(checkInRequest: CheckInRequest): void;
+    updateTicket(checkInRequest: CheckInRequest): Ticket;
```

Comment:
The return values should be wrapped in a Promise<>.

There should also be a definition for the initialize function:
```
initialize(config: ExternalConfiguration, http: IHttpDelegate): void;
```